requirements.txt

```
@@ -6,7 +6,7 @@ scipy>=0.16.0
 matplotlib>=2.0.0
 numpy>=1.16.0,!=1.19.0
 pillow
-h5py>=2.10
+h5py==3.0.0
```

```suggestion
h5py>=3.0.0
```
We probably don't want to require *exactly* 3.0.0. You also want to make the same change in https://github.com/gwastro/pycbc/blob/master/setup.py#L45